js-weather.js

```diff
@@ -0,0 +1,50 @@
+const escapeHtml = (str) => {
+  return str.toString().replace(/^\s+|\s+$/g,'');
+}
```

Comment:
(修正不要)
ちなみにアロー関数はさらに省略できます。できるだけ省略すべきというわけではないようですが。
```suggestion
const escapeHtml = str => str.toString().replace(/^\s+|\s+$/g,'');
```